tests/data/test_hdf5_daily_bars.py

```diff
@@ -27,3 +32,44 @@ def test_write_empty_country(self):
         assert_equal(reader.asset_start_dates, empty_dates)
         assert_equal(reader.asset_end_dates, empty_dates)
         assert_equal(reader.dates, empty_dates)
+
+    def test_multi_country_attributes(self):
```

Comment:
This test seems fine, but should we just add a test for `sessions` to `_DailyBarsTestCase` in `test_daily_bars`? Seems like we should have coverage for this for all of the readers.

Reply:
3366253af8ef34c3c338873c8b2132260e8cf35a